core/vibe/core/drivers/libevent2.d

```diff
@@ -77,6 +77,13 @@ version(Windows) {
 	}
 	else
 		import core.sys.freebsd.netinet.in_ : IP_ADD_MEMBERSHIP, IP_MULTICAST_LOOP;
+} else version (DragonFlyBSD) {
+	static if (__VERSION__ < 2077) {
```

Comment:
Shouldn't this be 2079 ? Since it's not merged yet, and 2078 was already released.

Reply:
You raising this question, made me think about this again. As this source is only going to be compiled/used when the changes have already landed in the compiler + druntime. It does make sense to even check the compiler version. I will remove this static if and just import core.sys.dragonflybsd.netinet.in_. There really is not need for the fall back.